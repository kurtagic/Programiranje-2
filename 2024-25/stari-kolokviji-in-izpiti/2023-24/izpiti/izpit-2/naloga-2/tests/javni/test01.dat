14
3 3 3 3 5 2 2 2 7 7 9 8 8 8
6
0 4 5 8 10 11
