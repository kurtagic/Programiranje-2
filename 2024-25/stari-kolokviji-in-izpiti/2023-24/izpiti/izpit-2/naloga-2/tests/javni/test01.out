[3, 3, 3, 3, 5, 2, 2, 2, 7, 7, 9, 8, 8, 8]
[12, 5, 6, 14, 9, 24]
111111
